### PR TITLE
Rgb to rgba

### PR DESCRIPTION
**Pull request by ucalyptus**
Lint errors resolved 99%

### CODE REVIEW

File: kornia/color/rgb.py [hunk 1]
```diff
@@ -2,6 +2,55 @@
...
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W).Got {}"
+                         .format(image.shape))
+
+    # add one channel
+    r, g, b = torch.chunk(image, image.shape[-3], dim=-3)
+    a = torch.Tensor(r.shape)
+    a[-1, 0, -1] = aval / 255
+    out: torch.Tensor = torch.cat([r, g, b, a], dim=-3)
```
Comment (edgarriba) [inline kornia/color/rgb.py]: wouldn't this be something like this?
```python
r, g, b = torch.chunk(image, chunks=3, dim=-3)
a: torch.Tensor = torch.full_like(r, fill_value= float(alpha_val) / 255)
return torch.cat([r, g, b, a], dim=-3)
```

File: kornia/color/rgb.py [hunk 2]
```diff
@@ -2,6 +2,55 @@
...
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(aval)
+        >>> output = rgba(input)  # 2x4x4x5
+
+   """
+    def __init__(self) -> None:
+
+        super(RgbToRgba, self).__init__()
+
+    def forward(self, image: torch.Tensor, aval: int) -> torch.Tensor:
```
Comment (edgarriba) [inline kornia/color/rgb.py]: variable name more explicit `aval` -> `alpha_val: float`

File: kornia/color/rgb.py [hunk 3]
```diff
@@ -2,6 +2,55 @@
...
+        return rgb_to_rgba(image, aval)
+
+
+def rgb_to_rgba(image: torch.Tensor, aval: int):
+
+    if not torch.is_tensor(image):
+
```
Comment (edgarriba) [inline kornia/color/rgb.py]: remove space

File: kornia/color/rgb.py (hunk 1 above)
Comment (ucalyptus) [reply kornia/color/rgb.py]: Yeah was unaware of full_like . Thanks

**Pull request closed without merging**